src/framework/update/imusesoundscheckupdateservice.h

```diff
@@ -45,7 +44,7 @@ class IMuseSoundsCheckUpdateService : MODULE_EXPORT_INTERFACE
 
     virtual Progress updateProgress() = 0;
 
-    virtual void openMuseHub() = 0;
+    virtual void openMuseHub(const std::vector<std::string>& actions) = 0;
```

Comment:
I'm not sure we need it in this interface... Perhaps it was originally intended to be reusable, but with the addition of the actions argument, it's unclear how to use it. We can make it a private method of MuseSoundsCheckUpdateScenario

Reply:
fixed